code/modules/admin/callproc/callproc.dm

```diff
@@ -136,72 +136,72 @@
-				if(isnull(current)) return CANCEL
+				if(isnull(current)) return CANCEL_N
 
 			if("type")
 				current = input("Select type for [arguments.len+1]\th argument") as null|anything in typesof(/obj, /mob, /area, /turf)
-				if(isnull(current)) return CANCEL
+				if(isnull(current)) return CANCEL_N
 
 			if("obj reference")
 				current = input("Select object for [arguments.len+1]\th argument") as null|obj in world
-				if(isnull(current)) return CANCEL
+				if(isnull(current)) return CANCEL_N
 
 			if("mob reference")
 				current = input("Select mob for [arguments.len+1]\th argument") as null|mob in world
-				if(isnull(current)) return CANCEL
+				if(isnull(current)) return CANCEL_N
 
 			if("area/turf reference")
 				current = input("Select area/turf for [arguments.len+1]\th argument") as null|area|turf in world
-				if(isnull(current)) return CANCEL
+				if(isnull(current)) return CANCEL_N
 
 			if("icon")
 				current = input("Provide icon for [arguments.len+1]\th argument") as null|icon
-				if(isnull(current)) return CANCEL
+				if(isnull(current)) return CANCEL_N
 
 			if("client")
 				current = input("Select client for [arguments.len+1]\th argument") as null|anything in GLOB.clients
-				if(isnull(current)) return CANCEL
+				if(isnull(current)) return CANCEL_N
 
 			if("mob's area")
 				var/mob/M = input("Select mob to take area for [arguments.len+1]\th argument") as null|mob in world
 				if(!M) return
 				current = get_area(M)
 				if(!current)
-					switch(alert("\The [M] appears to not have an area; do you want to pass null instead?",, "Yes", "Cancel"))
+					switch(alert("\The [M] appears to not have an area; do you want to pass null instead?",, "Yes", "CANCEL_N"))
 						if("Yes")
 							; // do nothing
-						if("Cancel")
-							return CANCEL
+						if("CANCEL_N")
+							return CANCEL_N
 
 			if("marked datum")
 				current = C.holder.marked_datum()
 				if(!current)
-					switch(alert("You do not currently have a marked datum; do you want to pass null instead?",, "Yes", "Cancel"))
+					switch(alert("You do not currently have a marked datum; do you want to pass null instead?",, "Yes", "CANCEL_N"))
 						if("Yes")
 							; // do nothing
-						if("Cancel")
-							return CANCEL
+						if("CANCEL_N")
+							return CANCEL_N
 
 			if("click on atom")
 				waiting_for_click = 1
-				add_verb(C, /client/proc/cancel_callproc_select)
-				to_chat(C, "Click an atom to select it. Click an atom then click 'cancel', or use the Cancel-Callproc-Select verb to cancel selecting a target by click.")
+				add_verb(C, /client/proc/CANCEL_N_callproc_select)
+				to_chat(C, "Click an atom to select it. Click an atom then click 'CANCEL_N', or use the CANCEL_N-Callproc-Select verb to CANCEL_N selecting a target by click.")
 				return WAITING
 
 		if(!done)
 			arguments += current
 
 	return DONE
 
-/client/proc/cancel_callproc_select()
-	set name = "Cancel Callproc Select"
+/client/proc/CANCEL_N_callproc_select()
```

Comment:
this looks like a lot of victims of massreplace that shouldn't have been changed, i think you forgot to use Match Case

Reply:
Yeah. Fix now